### ISSUE TITLE
Vogliamo puntare ai comandi di conversione, piuttosto che a pandoc?

### ISSUE THREAD

**Issue by danse** [italia/docs-italia-guide#8]
Se sei d'accordo proporrei una pull request riguardo alla parte di conversione in modo che punti ai comandi: https://github.com/italia/docs-italia-comandi-conversione#comandi-per-la-conversione

**Comment by alterat** [comment]
@danse 
Sono d'accordo, credo che sia giunto il momento di mettere un link ai comandi di conversione nella guida. 

Domanda: vogliamo eliminare i riferimenti a pandoc o vogliamo presentare i tuoi comandi di conversione assieme a pandoc?

**Comment by danse** [comment]
Fra l'altro @atorin se sei d'accordo migrerei tutte le info che hanno a che vedere con pandoc [nella sezione relativa della doc dei comandi](https://github.com/italia/docs-italia-comandi-conversione/blob/master/doc/comandi/pandoc.md), in modo da evitare duplicazioni

**Comment by danse** [comment]
Whoops, avevo la pagina aperta ed il tuo messaggio è apparso solo dopo che scrivessi il mio. Beh vedo che ci siamo anticipati a vicenda! :sweat_smile: 

Allora preparerò una migrazione dei contenuti e te la sottoporrò a breve

**Comment by danse** [comment]
@atorin ho migrato alcuni contenuti dalla guida alla doc dei comandi relativa a pandoc, puoi dare un'occhiata qui https://github.com/italia/docs-italia-comandi-conversione/blob/master/doc/comandi/pandoc.md, qualsiasi modifica è la benvenuta. il passaggio successivo sarà rimuovere i riferimenti a pandoc dalla guida e fare riferimento direttamente ai comandi ed al convertitore web

**Comment by danse** [comment]
credo ci sia tutto, fra #5 e https://github.com/italia/docs-italia-comandi-conversione/blob/master/doc/comandi/pandoc.md

**Comment by alterat** [comment]
@danse questa sezione andrebbe all'interno della Guida a Docs Italia (qui, per esempio: http://guida-docs-italia.readthedocs.io/it/latest/index/scrivere-un-documento.html#migrazione-su-docs-italia-di-documentazione-esistente)?

**Comment by danse** [comment]
quale sezione intendi, quella su pandoc?

**Comment by alterat** [comment]
@danse 
[Questo file](https://github.com/italia/docs-italia-comandi-conversione/blob/master/doc/comandi/pandoc.md) intendi copiarlo pari pari all'interno della Guida a Docs Italia?

**Comment by danse** [comment]
no, quel file lo lascerei lì, sarebbe raggiungibile tramite questo https://github.com/italia/docs-italia-comandi-conversione#maggiori-informazioni indice, come gli altri comandi. Quindi gli utenti a partire dalla guida farebbero il seguente percorso:

```
-> conversione
 -> convertitore web
 -> comandi di conversione
   -> pandoc
   -> converti
   -> pandoc-to-sphinx
   -> ...
```

**Comment by alterat** [comment]
OK... quindi rimane comunque la questione di aggiornare la Guida mettendo i link al nuovo convertitore e ai tuoi comandi, giusto?

Come vogliamo fare?

**Comment by danse** [comment]
riguardo a quello ho aggiunto commit alla pull request che er

**Comment by danse** [comment]
riguardo a quello ho aggiunto alcuni commit alla pull request #5 che era già aperta
